### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Edit Configuration popup_Screen Reader: Jaws is not reading anything when “Submit spark application” is selected.

### ISSUE THREAD

**Issue by v-lasiva** [microsoft/azure-tools-for-java#3620]
[Check out Accessibility Insights! ](url)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #MAS1.3.1; #ScreenReader; #Jaws; #C+E-H; #JavatoolingforIntelliJ/Eclipse;

**Environment Details:**
URL: https://www.jetbrains.com/idea/download/#section=windows
Version: 2019.2.2
Build: 192.6603.28
Screen Reader: JAWS.
MAS Violated: MAS1.3.1

**Repro Steps:**
1.Turn on screen reader.
2.Download IntelliJ application from the URL:  https://www.jetbrains.com/idea/download/#section=windows .
3.Launch IntelliJ IDEA.
4.Tab till Create New Project button and press enter
5.Tab till Azure Spark/HDInsight-->Spark Project Scala
6.Tab till Build Tools-->Maven
7.Tab till Next button and press enter
8.Tab till finish button and press enter
9.Navigate to project name-->src-->main-->scala-->project name and open the context menu using keyboard key.
10.Tab till Submit Spark Application and press enter
11.Verify whether jaws is reading the complete information that is present in the pop-up displayed on the screen.

**Actual Result:**
Jaws is not reading anything when “Submit spark application” is selected and Keyboard functionality is not proper when screen reader is on.

**Expected Result:**
Jaws should read the complete information that is present in the pop-up when “Submit spark application” is selected and Keyboard functionality should be proper even when screen reader is on.

**User Impact:**
Screen Reader users will face difficulty in knowing what information is present in the pop-up, If Jaws doesn't read anything.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug_Jaws is not reading anything when edit config popup is opened.webm.pptx](https://github.com/microsoft/azure-tools-for-java/files/3656040/Bug_Jaws.is.not.reading.anything.when.edit.config.popup.is.opened.webm.pptx)

**Comment by t-rufang** [comment]
Note: Screen reader is not reading anything in the run configuration window
![image](https://user-images.githubusercontent.com/32627233/65663479-850e3700-e069-11e9-9e73-f92237c649fe.png)

**Comment by t-rufang** [comment]
Fixed in #3687

**Comment by v-lasiva** [comment]
Regressed the issue in the updated version of IntelliJ having the following details,
Version: 2019.2.2

Now Jaws reading the complete information that is present in the pop-up when “Submit spark application” is selected and Keyboard functionality is also proper when screen reader is on.
As the issue is resolved, Hence closing the bug.
Added attachment for reference.

[Bug_Jaws is reading complete data_Issue fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3850979/Bug_Jaws.is.reading.complete.data_Issue.fixed.pptx)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749720
